src/compute/compute.py

```diff
@@ -187,7 +187,7 @@ def submit_job_task(auth_header, system_name, system_addr, job_file, job_dir, ta
 
         jobid = extract_jobid(outlines)
 
-        msg = {"result" : "Job submitted", "jobid" : jobid, "jobfile" : f"{job_dir}/{job_file['filename']}"}
+        msg = {"result" : "Job submitted", "jobid" : jobid, "job_file" : f"{job_dir}/{job_file['filename']}"}
```

Comment:
I assume it works and I don't see anything wrong with it but why not omit it and let it be updated in `get_slurm_files`, like you do in `submit_job_path_task`? I suggest it mostly for consistency.

Reply:
Good observation. Will be fixed.